### PR TITLE
Add missing Parser API in dts for javascript runtime

### PR DESCRIPTION
**Pull request by HerrCai0907**

### CODE REVIEW

**Comment by mike-lischke** [comment]
Obsolete with my patch #4403. Can be closed.

**Comment by mike-lischke** [comment]
@parrt, this can also be closed.

**Comment by ericvergnaud** [comment]
@HerrCai0907 can you reopen this ?

**Comment by parrt** [comment]
I am wondering if we should break up the big patch into more bite sized
portions for easy review and decision making?

**Comment by mike-lischke** [comment]
@parrt yes, @ericvergnaud requested that already and I started to split the patch.

File: runtime/JavaScript/src/antlr4/Parser.d.ts [hunk 1]
```diff
@@ -41,5 +41,7 @@ export declare class Parser extends Recognizer<Token> {
     setTokenStream(input: TokenStream): void;
     notifyErrorListeners(msg: string, offendingToken: Token, err: RecognitionException | undefined): void;
     getCurrentToken(): Token;
-    
+    addParseListener(listener: any): void;
+    removeParseListener(listerner: any): void;
```
Comment (ericvergnaud) [inline runtime/JavaScript/src/antlr4/Parser.d.ts:45]: There's a typo.

**Pull request closed without merging**